src/oc_erchef/apps/oc_chef_wm/src/chef_wm_stats.erl

```diff
@@ -0,0 +1,59 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%%% @doc
+%%% REST resource for reporting chef server stats
+%%% @end
+%% Copyright 2017 Chef Software, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+
+-module(chef_wm_stats).
+
+-ifdef(TEST).
+-compile(export_all).
+-endif.
+
+-export([init/1,
+         allowed_methods/2,
+         content_types_provided/2,
+         to_json/2,
+         to_text/2]).
+
+-include_lib("webmachine/include/webmachine.hrl").
+-define(A2B(X), erlang:atom_to_binary(X, utf8)).
+
+init(_Any) ->
+    {ok, <<"{}">>}.
```

Comment:
What's the use of passing `<<"{}">>` around as `State`? (Am I missing something?)

Reply:
Not sure, I just copy pasted from elsewhere. Will take a closer look here

Reply:
Cleaned up.